### ISSUE TITLE
Twisted Edge Extreme Snowboarding no audio

### ISSUE THREAD

**Issue by loganmc10** [mupen64plus/mupen64plus-core#285]
There is a cheat in Project64 that makes it work:

```
"exhalatio audio hack",81051840 AD99
```

And in the AziAudio plugin it was fixed this way:
https://github.com/Azimer/AziAudio/pull/181/commits/7140f74be33615b4f9c62164e5e7e4d261cc6e48

I tried both methods in mupen64plus without success, still no audio

**Comment by loganmc10** [comment]
It looks like ```push_audio_samples_via_audio_plugin``` only gets called 5 times, right at the beginning of the game, and then never again, so the first thing would be to figure out why the audio samples aren't being pushed

**Comment by loganmc10** [comment]
If it's any kind of a hint, modifying CountPerScanline will affect how many times ```push_audio_samples_via_audio_plugin``` is called. At 1500 (the default), it is called 5 times, at 2200 it is called 4 times, at 1200 it is called 12 times.

**Comment by loganmc10** [comment]
Also, if I let it print out the TLB exceptions I see this:

```
tlb exception !!! @ 4000, 1, add:80001844
```

**Comment by loganmc10** [comment]
@Azimer @cxd4 do you know what was done in Project64 to fix this? From my reading, it seems like it was fixed somewhat recently (2014+).

I am reading all kinds of stuff about the RSP not getting the audio tasks, but I'm not able to find any information about what resolved the issue. It seems we are having the same issue.

**Comment by Azimer** [comment]
Twisted Edge does not use RSP for audio.  I can't comment on the Project64 fix, however in 1.6 I was able to get a consistent stream of samples after tweaking audio timing.
